sui/src/rest_gateway/requests.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use schemars::JsonSchema;
+use serde::Deserialize;
+use serde::Serialize;
+use serde_with::base64::Base64;
+use serde_with::serde_as;
+
+/**
```

Comment:
Shouldn't we use the Doc comments `///` instead of `/** **/` here? https://doc.rust-lang.org/rust-by-example/meta/doc.html#doc-comments

Reply:
I think this is how the rest api docs likes it (I am not very sure)? @arun-koshy ?

Reply:
Just tested and actually both work when generating the open api descriptions in dropshot. We can switch to `///` as that seems to be the canonical way in rust. I was just following the format in the [dropshot examples](https://docs.rs/crate/dropshot/latest/source/examples/file_server.rs)

Reply:
yes in general we should prefer `///` for doc comments